--- C:/Users/Reinhardt/Downloads/pypdfocr/pypdfocr-0.9.1/pypdfocr/pypdfocr_pdffiler.py	Thu Feb 18 22:02:07 2016
+++ C:/Users/Reinhardt/Downloads/pypdfocr/pypdfocr-0.9.1/pypdfocr/pypdfocr_pdffiler_regex.py	Tue Aug 08 12:28:31 2017
@@ -53,7 +53,8 @@
         for folder,strings in self.filer.folder_targets.items():
             for s in strings:
                 logging.debug("Checking string %s" % s)
-                if s in searchText:
+                # if s in searchText:
+				if re.search(s, searchText, flags=re.M):
                     logging.info("Matched keyword '%s'" % s)
                     return folder
         # No match found, so return 
